server/index.js

```diff
@@ -10,85 +10,91 @@ const path = require('path');
 const fs = require('bfile');
 const { execSync } = require('child_process');
 const os = require('os');
-const Config = require('bcfg');
-const logger = require('./logger');
-const Blgr = require('blgr');
+const { createLogger } = require('./logger');
+const Logger = require('blgr');
 const chokidar = require('chokidar');
 
 const webpackArgs = [];
 
 let poll = false;
 // If run from command line, parse args
 if (require.main === module) {
-  // setting up webpack configs
-  // use default/base config for dev
-  if (
-    process.argv.indexOf('--dev') >= 0 ||
-    process.env.NODE_ENV === 'development'
-  ) {
-    webpackArgs.push(
-      '--config',
-      path.resolve(__dirname, '../configs/webpack.config.js')
-    );
-  } else {
-    // otherwise use prod config
-    webpackArgs.push(
-      '--config',
-      path.resolve(__dirname, '../configs/webpack.prod.js')
-    );
-  }
+  (async function() {
+    const logger = createLogger();
+    await logger.open();
+
+    // setting up webpack configs
+    // use default/base config for dev
+    if (
+      process.argv.indexOf('--dev') >= 0 ||
+      process.env.NODE_ENV === 'development'
+    ) {
+      webpackArgs.push(
+        '--config',
+        path.resolve(__dirname, '../configs/webpack.config.js')
+      );
+    } else {
+      // otherwise use prod config
+      webpackArgs.push(
+        '--config',
+        path.resolve(__dirname, '../configs/webpack.prod.js')
+      );
+    }
 
-  // environment specific `watch` args
-  if (process.argv.indexOf('--watch-poll') >= 0) {
-    poll = true;
-    webpackArgs.push('--watch', '--env.dev', '--env.poll');
-  } else if (process.argv.indexOf('--watch') >= 0) {
-    webpackArgs.push('--watch', '--env.dev');
-  }
+    // environment specific `watch` args
```

Comment:
Ideally we check the os here and push options accordingly instead of the user having to know which flag to pass based on knowing their os